src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3997,6 +4121,130 @@ namespace ZoneTempPredictorCorrector {
 				SNLoad = 0.0;
 			}
 
+			//Hybrid modeling start: Added by Sang Hoon Lee May 2015
+			if (( HybridModelZone( ZoneNum ).InternalThermalMassCalc || HybridModelZone( ZoneNum ).InfiltrationCalc ) && !WarmupFlag && !DoingSizing ) {
+
+				Zone( ZoneNum ).ZoneMeasuredTemperature = GetCurrentScheduleValue( HybridModelZone( ZoneNum ).ZoneMeasuredTemperatureSchedulePtr );
+
+				// prepare start and end date for Hybrid Modeling
```

Comment:
Also, looks like this doesn't allow for leap years.  There should already be some utility functions in EnergyPlus to convert these start/end dates.  Look in the weather manager or elsewhere.